django/core/handlers/base.py

```diff
@@ -141,11 +141,6 @@ def get_response(self, request):
 
         response._closable_objects.append(request)
 
-        # If the exception handler returns a TemplateResponse that has not
-        # been rendered, force it to be rendered.
-        if not getattr(response, 'is_rendered', True) and callable(getattr(response, 'render', None)):
```

Comment:
I think we should leave these lines for now. It's a deprecated code path requiring use of `MIDDLEWARE_CLASSES` but `self.handle_uncaught_exception` on line 140 might return a `TemplateResponse` in which case things crash without this.